.github/workflows/package-apisix-dashboard.yml

```diff
@@ -1,4 +1,4 @@
-name: package apisix-dashabord
+name: Package apisix-dashabord
```

Comment:
Is this change necessary for deb support?

Reply:
Please fix other files - not just the file we pointed out

Reply:
@EnableAsync 
Please don't mark the conversation as resolved by yourself. It makes review harder. And obviously, this conversation is not resolved.